src/main/java/mate/academy/connection/ConnectionUtils.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy.connection;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+import mate.academy.exception.DataProcessingException;
+
+public class ConnectionUtils {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/test";
+    private static final Properties DB_PROPERTIES;
+
+    static {
+        DB_PROPERTIES = new Properties();
+        DB_PROPERTIES.put("user", "root");
+        DB_PROPERTIES.put("password", "4321");
```

Comment:
Storing database credentials directly in the code is not a secure practice. Consider externalizing these properties to a configuration file or environment variables to enhance security.